cmd/experimental/kjobctl/pkg/cmd/testing/fake.go

```diff
@@ -65,6 +71,16 @@ func (cg *TestClientGetter) WithRESTMapper(mapper meta.RESTMapper) *TestClientGe
 	return cg
 }
 
+func (cg *TestClientGetter) WithRESTConfig(config *rest.Config) *TestClientGetter {
+	cg.restConfig = config
+	return cg
+}
+
+// ToRESTConfig is used to get ClientConfigVal from a TestFactory
+func (cg *TestClientGetter) ToRESTConfig() (*rest.Config, error) {
+	return cg.restConfig, nil
+}
```

Comment:
IIRC `genericclioptions.RESTClientGetter` already have ToRESTConfig method.

Reply:
Yes, however, it seems that the restConfig property isn't writable. Therefore, I've overwritten this method for testing.